### PR TITLE
fix: firebolt query was ignoring access_token and allways used user/pass

### PR DESCRIPTION
**Pull request by yuryfirebolt**

### CODE REVIEW

File: src/firebolt_cli/query.py [hunk 1]
```diff
@@ -172,31 +172,14 @@ def query(**raw_config_options: str) -> None:
...
 
-    # Decide whether to store the value as engine_name or engine_url
-    # '.' symbol should always be in url and cannot be in engine_name
-    engine_name, engine_url = None, None
-
+    # if engine_name is not set, use default engine
     if raw_config_options["engine_name"] is None:
         rm = construct_resource_manager(**raw_config_options)
-        engine_name = get_default_database_engine(
+        raw_config_options["engine_name"] = get_default_database_engine(
```
Comment (stepansergeevitch) [inline src/firebolt_cli/query.py:178]: Isn't it more efficient to put url here rather than name?
We won't have to resolve name later in this case

**Review by stepansergeevitch — COMMENTED** [review]
lgtm except for a single comment

**Review by ptiurin — APPROVED** [review]
Lgtm, agree with Stepan's point.

File: src/firebolt_cli/query.py (hunk 1 above)
Comment (yuryfirebolt) [reply src/firebolt_cli/query.py:178]: It is, changed it

**Pull request merged**